app/models/flood_risk_engine/comment.rb

```diff
@@ -0,0 +1,6 @@
+module FloodRiskEngine
+  class Comment < ActiveRecord::Base
+    belongs_to :commentable, polymorphic: true
+    belongs_to(:user) if defined?(User)
```

Comment:
this is not the way to do it then. the belongs_to(:user)  should be defined in the BO app only